com.github.rhyme_player.rhyme.yml

```diff
@@ -0,0 +1,66 @@
+app-id: com.github.rhyme_player.rhyme
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "20.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does wayland actually work here? (try running on the commandline with --nosocket=x11 on a wayland session). If so you should replace x11 with fallback-x11 to use wayland by default.

Reply:
It's Electron. It supports native Wayland, but it's not enabled by default yet.

Reply:
"not enabled by default" is "not supported" in my book. This should be removed.

Reply:
Yes, unless you are actually changing electron to support it yourself the permission should be removed for now.